CMakeLists.txt

```diff
@@ -1,479 +1,44 @@
-eth_format_option(ROCKSDB)
-eth_format_option(GUI)
-eth_format_option(TESTS)
-eth_format_option(ROCKSDB)
-eth_format_option(TOOLS)
-eth_format_option(ETHKEY)
-eth_format_option(ETHASHCL)
-eth_format_option(JSCONSOLE)
-eth_format_option(OLYMPIC)
-eth_format_option(SERPENT)
-
-# Default CMAKE_BUILD_TYPE accordingly.
-set(CMAKE_BUILD_TYPE CACHE STRING ${D_CMAKE_BUILD_TYPE})
-
-# Default TARGET_PLATFORM to ${CMAKE_SYSTEM_NAME}
-# change this once we support cross compiling
-set(TARGET_PLATFORM CACHE STRING ${CMAKE_SYSTEM_NAME})
-if ("x${TARGET_PLATFORM}" STREQUAL "x")
-	set(TARGET_PLATFORM ${CMAKE_SYSTEM_NAME})
-endif ()
-
-include(EthDependencies)
-
-configureProject()
-
-message("------------------------------------------------------------------------")
-message("--                  CMake Version                            ${CMAKE_VERSION}")
-message("-- CMAKE_BUILD_TYPE Build type                               ${CMAKE_BUILD_TYPE}")
-message("-- TARGET_PLATFORM  Target platform                          ${TARGET_PLATFORM}")
-message("-- BUNDLE           Build bundle                             ${BUNDLE}")
-message("--------------------------------------------------------------- features")
-message("--                  Chromium support                         ${ETH_HAVE_WEBENGINE}")
-message("--                  Hardware identification support          ${CPUID_FOUND}")
-message("--                  HTTP Request support                     ${CURL_FOUND}")
-message("-- VMTRACE          VM execution tracing                     ${VMTRACE}")
-message("-- PROFILING        Profiling support                        ${PROFILING}")
-message("-- FATDB            Full database exploring                  ${FATDB}")
-message("-- USENPM           Javascript source building               ${USENPM}")
-message("-- ROCKSDB          Prefer rocksdb to leveldb                ${ROCKSDB}")
-message("-- OLYMPIC          Default to the Olympic network           ${OLYMPIC}")
-message("------------------------------------------------------------- components")
-message("-- MINER            Build miner                              ${MINER}")
-message("-- ETHKEY           Build wallet tools                       ${ETHKEY}")
-message("-- TOOLS            Build basic tools                        ${TOOLS}")
-message("-- SERPENT          Build Serpent language components        ${SERPENT}")
-message("-- GUI              Build GUI components                     ${GUI}")
-message("-- TESTS            Build tests                              ${TESTS}")
-message("-- ETHASHCL         Build OpenCL components                  ${ETHASHCL}")
-message("-- JSCONSOLE        Build with javascript console            ${JSCONSOLE}")
-message("-- EVMJIT           Build LLVM-based JIT EVM                 ${EVMJIT}")
-message("------------------------------------------------------------------------")
-message("")
-
-set(CMAKE_THREAD_LIBS_INIT pthread)
+project(cpp-ethereum VERSION "0.9.91")
```

Comment:
That's for checking that. Have you checked also if the old `ethereum_VERSION` and `ethereum_SOURCE_DIR` have been replaced also?

It's umbrella project. It's optional. Does anyone care about its version? Does libethereum and AZ have theirs own versions? Should you bump it depending on what?

Maybe it's reasonable to remove the version here.